vertexai/snippets/src/main/java/vertexai/gemini/samples/MultimodalChat.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package vertexai.gemini.samples;
+
+import com.google.cloud.vertexai.VertexAI;
+import com.google.cloud.vertexai.generativeai.preview.*;
+import com.google.cloud.vertexai.v1beta1.GenerateContentResponse;
+
+import java.util.Base64;
+
+public class MultimodalChat {
+
+    private static final String PROJECT_ID = System.getenv("GOOGLE_CLOUD_PROJECT");
+    private static final String LOCATION = "us-central1";
+    private static final String MODEL_NAME = "gemini-ultra-vision";
+
+    private static final String DATA_IMAGE_PNG_BASE_64 =
```

Comment:
Applies across files: You could leave this empty in the sample and add a comment describing the variable.

Reply:
The test would fail without the data?

Reply:
I embedded the value in the test file and removed it from the sample.